sigs.yaml

```diff
@@ -3245,6 +3245,44 @@ workinggroups:
     liaison:
       github: cblecker
       name: Christoph Blecker
+- dir: wg-lts
+  name: LTS
+  mission_statement: >
+    Define LTS
+
+  stakeholder_sigs:
```

Comment:
IMO SRC should be a stakeholder here but we can deal with that later as part of the WG.

Reply:
I don't believe the generator/validation allows for anything other than valid SIGs here. I agree that committees could be stakeholders, but I don't think that has been a thin in the past? 

@kubernetes/steering-committee  what do you think should be done here?

Reply:
@ritazh I added the security response committee here as a label to capture that, for now.

Reply:
@palnabarun possibly, I was holding off adding any additional SIGs given the other discussions that have occurred but I will add this one because I think it's likely to be involved regardless.